### PR TITLE
Staging v1.9.3

### PR DESCRIPTION
**Pull request by fernandofloresg**
##### SUMMARY

Merge release branch into branch 1.9.3 to create the tag **v1.9.3**. This time this will be the only time we can do a review in the process since this won't be merged into main nor dev.

##### ISSUE TYPE

- Release Pull Request

### CODE REVIEW

File: docs/source/resources/releases_maintenance.rst [hunk 1]
```diff
@@ -6,39 +6,95 @@
 Releases and maintenance
 ========================
 
-This table describes the collections release dates, dependency versions and End of Life dates (EOL).
+This section describes the collections release dates, dependency versions and End of Life dates (EOL)
```
Comment (ketankelkar) [inline docs/source/resources/releases_maintenance.rst:9]: you've got an extra `and` in there

```suggestion
This section describes the collections release dates, dependency versions, End of Life dates (EOL)
```

File: docs/source/resources/releases_maintenance.rst [hunk 2]
```diff
@@ -6,39 +6,95 @@
...
+
 These are the component versions available when the collection was made generally available (GA). The underlying
 component version is likely to change as it reaches EOL, thus components must be a version that is
 currently supported.
 
 For example, if a collection releases with a minimum version of ``ansible-core`` 2.14.0 (Ansible 7.0) and later this
 enters into EOL, then a newer supported version of ``ansible-core`` (Ansible) must be selected. When choosing a newer
 ``ansible-core`` (Ansible) version, review the `ansible-core support matrix`_ to select the appropriate dependencies.
-This is important to note, different releases of ``ansible-core`` can require newer controller and managed node
+This is important to note, different releases of ``ansible-core`` can require newer control node and managed node
```
Comment (ketankelkar) [inline docs/source/resources/releases_maintenance.rst:70]: i'm curious as to the reasoning behind this change from controller to control node, personally i find it easier to use 'controller' and 'target node' to keep the two straight in my head (as opposed to control node and target or managed node)

**Review by ketankelkar — APPROVED** [review]
left a few comments above, but nothing NEEDs to change, so approved. nice work here!!

File: docs/source/resources/releases_maintenance.rst (hunk 2 above)
Comment (fernandofloresg) [reply docs/source/resources/releases_maintenance.rst:70]: Not sure why @ddimatos changed this, reviewing the Ansible docs I see it mentioned as **controller** but when talking about architecture is mentioned as **control-node** [here](https://www.ansible.com/how-ansible-works/). and **managed node** seems to be the preferred word.

File: CHANGELOG.rst [hunk 3]
```diff
@@ -4,6 +4,25 @@ ibm.ibm\_zos\_core Release Notes
 
 .. contents:: Topics
 
+v1.9.3
+======
+
+Release Summary
+---------------
+
+Release Date: '2024-10-23'
```
Comment (richp405) [inline CHANGELOG.rst:13]: would this be 10-23, which was last week?

File: docs/source/resources/releases_maintenance.rst (hunk 2 above)
Comment (richp405) [reply docs/source/resources/releases_maintenance.rst:70]: controller implies human (one who controls).  control node implies a machine.

**Review by richp405 — APPROVED** [review]
A minor date issue, and agreeing with D on the 'control node' instead of 'controller'.

File: CHANGELOG.rst (hunk 3 above)
Comment (fernandofloresg) [reply CHANGELOG.rst]: updated it

**Pull request merged**